changes_test.go

```diff
@@ -2060,16 +2090,53 @@ func (s *changesSuite) TestChangeDescriptions(c *gc.C) {
 			expectedChanges: []string{
 				"upload charm cs:keystone",
 				"deploy application keystone using cs:keystone",
-				"add new machine $0",
-				"add unit keystone/$0 to 0/lxd/$1 to satisfy [lxd:mysql]",
-				"add lxd container $0/lxd/$0 on new machine $0",
-				"add lxd container $1/lxd/$0 on new machine $1",
-				"add unit mysql/$1 to $0/lxd/$0",
-				"add unit mysql/$2 to $1/lxd/$0",
-				"add lxd container $0/lxd/$1 on new machine $0",
-				"add lxd container $1/lxd/$1 on new machine $1",
-				"add unit keystone/$1 to $0/lxd/$1 to satisfy [lxd:mysql]",
-				"add unit keystone/$2 to $1/lxd/$1 to satisfy [lxd:mysql]",
+				"add new machine 1 (bundle machine 0)",
+				"add unit keystone/0 to 0/lxd/1 to satisfy [lxd:mysql]",
+				"add lxd container 1/lxd/0 on new machine 1",
+				"add lxd container 2/lxd/0 on new machine 2",
+				"add unit mysql/1 to 1/lxd/0",
+				"add unit mysql/2 to 2/lxd/0",
+				"add lxd container 1/lxd/1 on new machine 1",
+				"add lxd container 2/lxd/1 on new machine 2",
+				"add unit keystone/1 to 1/lxd/1 to satisfy [lxd:mysql]",
+				"add unit keystone/2 to 2/lxd/1 to satisfy [lxd:mysql]",
+			},
+		}, {
+			description: "lxd container sequence",
+			bundleContent: `
+                applications:
+                    mysql:
+                        charm: cs:mysql
+                        num_units: 1
+                    keystone:
+                        charm: cs:keystone
+                        num_units: 1
+                        to: ["lxd:mysql"]
+            `,
+			existingModel: &bundlechanges.Model{
+				Applications: map[string]*bundlechanges.Application{
+					"mysql": &bundlechanges.Application{
+						Charm: "cs:mysql",
+						Units: []bundlechanges.Unit{
+							{"mysql/0", "0/lxd/0"},
+						},
+					},
+				},
+				Machines: map[string]*bundlechanges.Machine{
+					// We don't actually look at the content of the machines
+					// for this test, just the keys.
+					"0": nil, "0/lxd/0": nil,
+				},
+				Sequence: map[string]int{
+					"application-mysql": 1,
+					"machine":           1,
+					"machine-0/lxd":     2,
```

Comment:
Bugger, I think I need to rewrite this sequence name in the 1.25 upgrade.

Reply:
Nope.